### ISSUE TITLE
Node course: Update links to other course lessons

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28529]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

With the updated Node course, a couple of links need their hrefs updating to point to the correct lessons.

## Acceptance criteria

- [ ] `nodeJS/apis/project_blog_api.md`
  - [ ] Href for the deployment lesson in the last assignment step is replaced with `https://www.theodinproject.com/lessons/node-path-nodejs-deployment`
- [ ] `nodeJS/express/deployment.md`
  - [ ] href for the mini message board project in the first sentence of the assignment is replaced with `https://www.theodinproject.com/lessons/node-path-nodejs-mini-message-board`
 
Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-nodejs-deployment#assignment && https://www.theodinproject.com/lessons/node-path-nodejs-blog-api

**Comment by nolanborz** [comment]
This would be my first contribution but I checked it out and I should be able to switch those 2 hrefs. My Discord is nolan#4041

Currently on the 'Using Git in the Real World' lesson in the Ruby track.

**Comment by MaoShizhong** [comment]
Go for it. If you have any questions, feel free to comment here or in your PR if relevant.

**Comment by nolanborz** [comment]
Got those 2 links updated and sent the pull request in. The instructions for the whole process were clear and it went smoothly.
